src/components/DatePicker/index.ios.js

```diff
@@ -3,14 +3,15 @@ import React from 'react';
 import {Button, View, Keyboard} from 'react-native';
 import RNDatePicker from '@react-native-community/datetimepicker';
 import moment from 'moment';
-import _ from 'underscore';
+import _, {compose} from 'underscore';
 import TextInput from '../TextInput';
 import withLocalize, {withLocalizePropTypes} from '../withLocalize';
 import Popover from '../Popover';
 import CONST from '../../CONST';
 import styles from '../../styles/styles';
 import themeColors from '../../styles/themes/default';
 import {propTypes, defaultProps} from './datepickerPropTypes';
+import withKeyboardState from '../withKeyboardState';
```

Comment:
We also need to define the props from `withKeyboardState`. You can get the props-types with `keyboardStatePropTypes`.